addons/payment/tests/test_payment_transaction.py

```diff
@@ -2,9 +2,13 @@
 
 from odoo.tests import tagged
 
+from odoo.tools import mute_logger
```

Comment:
These two can be in a single block, put in alphabetical order =]

Reply:
Done :)